src/cloud-api-adaptor/podvm/Dockerfile.podvm_binaries.fedora

```diff
@@ -36,6 +37,7 @@ ENV GUEST_COMPONENTS_REPO=${GUEST_COMPONENTS_REPO}
 ENV ATTESTER=${ATTESTER}
 ENV ARCH=${ARCH}
 ENV DEFAULT_AGENT_POLICY_FILE=${DEFAULT_AGENT_POLICY_FILE}
+ENV IMAGE_NAME=${IMAGE_NAME}
```

Comment:
If I understand correctly, are we going to drop the scripts in `/podvm/Makefile.inc`?
```
ifndef IMAGE_NAME
COMMIT := $(shell	commit=$$(git describe --match '' --dirty --always) && \
					if [ -n "$$(git ls-files --other --exclude-per-directory .gitignore "$(FILES_DIR)")" ]; then \
						commit="$${commit%-dirty}-dirty"; \
					fi && \
	                echo "$$commit")
ifndef COMMIT
$(error Failed to derive an image name. Explicitly define IMAGE_NAME)
endif
IMAGE_NAME := $(IMAGE_PREFIX)-$(COMMIT)-$(DEB_ARCH)
endif
```

Reply:
well, they will still be there, but we will go around them in an mkosi built, by setting IMAGE_NAME.